v1/lib/server/utils.js

```diff
@@ -4,12 +4,12 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 const cssnano = require('cssnano');
 const autoprefixer = require('autoprefixer');
 const postcss = require('postcss');
 const path = require('path');
 const escapeStringRegexp = require('escape-string-regexp');
+const siteConfig = require('../../website/siteConfig.js');
```

Comment:
This broke docusaurus completely, see #1149

Reply:
Yeah, that doesn't look right. 

@yangshun @endiliey -- I think we need to fix this or revert this specific diff and push out a hotfix.

Reply:
https://github.com/facebook/Docusaurus/pull/1150

Reply:
Yeah this PR broke the release. My bad to overlook this. Will release a hotfix in ~1 hour.